WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.kt

```diff
@@ -0,0 +1,901 @@
+import org.wordpress.android.WordPress.Companion.getBitmapCache
+import org.wordpress.android.analytics.AnalyticsTracker
+import org.wordpress.android.analytics.AnalyticsTracker.Stat
+import org.wordpress.android.fluxc.Dispatcher
+import org.wordpress.android.fluxc.action.AccountAction
+import org.wordpress.android.fluxc.generated.AccountActionBuilder
+import org.wordpress.android.fluxc.store.AccountStore
+import org.wordpress.android.fluxc.store.AccountStore.AccountUsernameActionType
+import org.wordpress.android.fluxc.store.AccountStore.OnAccountChanged
+import org.wordpress.android.fluxc.store.AccountStore.OnUsernameChanged
+import org.wordpress.android.fluxc.store.AccountStore.PushAccountSettingsPayload
+import org.wordpress.android.fluxc.store.AccountStore.PushUsernamePayload
+import org.wordpress.android.login.LoginBaseFormFragment
+import org.wordpress.android.login.widgets.WPLoginInputRow
+import org.wordpress.android.ui.FullScreenDialogFragment
+import org.wordpress.android.ui.FullScreenDialogFragment.OnShownListener
+import org.wordpress.android.ui.RequestCodes
+import org.wordpress.android.ui.accounts.UnifiedLoginTracker
+import org.wordpress.android.ui.photopicker.MediaPickerConstants
+import org.wordpress.android.ui.photopicker.MediaPickerLauncher
+import org.wordpress.android.ui.photopicker.PhotoPickerActivity.PhotoPickerMediaSource
+import org.wordpress.android.ui.prefs.AppPrefsWrapper
+import org.wordpress.android.ui.reader.services.update.ReaderUpdateLogic.UpdateTask
+import org.wordpress.android.ui.reader.services.update.ReaderUpdateServiceStarter
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.MediaUtils
+import org.wordpress.android.util.StringUtils
+import org.wordpress.android.util.ToastUtils
+import org.wordpress.android.util.WPAvatarUtils
+import org.wordpress.android.util.WPMediaUtils
+import org.wordpress.android.util.extensions.getColorFromAttribute
+import org.wordpress.android.util.extensions.redirectContextClickToLongPressListener
+import org.wordpress.android.util.image.ImageManager
+import org.wordpress.android.util.image.ImageType
+import org.wordpress.android.widgets.WPTextView
+import java.io.File
+import java.net.URI
+import java.net.URISyntaxException
+import java.util.EnumSet
+import java.util.Locale
+import javax.inject.Inject
+
+class SignupEpilogueFragment : LoginBaseFormFragment<SignupEpilogueListener?>(),
+    FullScreenDialogFragment.OnConfirmListener, FullScreenDialogFragment.OnDismissListener,
+    OnShownListener {
+    private lateinit var mEditTextDisplayName: EditText
+    private lateinit var mEditTextUsername: EditText
+    private var mDialog: FullScreenDialogFragment? = null
+    private var mSignupEpilogueListener: SignupEpilogueListener? = null
+
+    private lateinit var mHeaderAvatarAdd: ImageView
+    private var mDisplayName: String? = null
+    private lateinit var mEmailAddress: String
+    lateinit var mPhotoUrl: String
```

Comment:
I think we could make this variable `private` (it was `protected` in Java)
```suggestion
    private lateinit var mPhotoUrl: String
```